gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/PathBasedHivePartitionExtendedFactory.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gobblin.data.management.copy.hive;
+
+import java.util.Properties;
+
+import com.typesafe.config.Config;
+
+import gobblin.util.ConfigUtils;
+import gobblin.annotation.Alias;
+
+/**
+ * A path based specific filter factory for generation of {@link PathBasedPartitionFilter}
+ */
+@Alias("PathPartition")
+public class PathBasedHivePartitionExtendedFactory implements HivePartitionExtendedFilterFactory {
```

Comment:
`PathBasedHivePartitionExtendedFactory` -> `PathBasedHivePartitionFilterFactory` (mostly meaning mention in the name it is a filter).